bootstrap/config/base/kfctl_default.yaml

```diff
@@ -40,3 +41,6 @@ spec:
       - name: reportUsage
         value: "true"
         initRequired: true
+    admission-webhook-bootstrap:
```

Comment:
you can do the same thing if you add nameprefix admission-webfix- in kustomization.yaml. Though it adds it to all resource names.